tools/transform.cpp

```diff
@@ -568,13 +591,22 @@ static void calculateAndInitConstants(Transform &t) {
                    isCast(ConversionOp::Ptr2Int, i)) {
           max_alloc_size = max_access_size = cur_max_gep = UINT64_MAX;
           has_int2ptr |= isCast(ConversionOp::Int2Ptr, i) != nullptr;
-          has_ptr2int |= isCast(ConversionOp::Ptr2Int, i) != nullptr;
+          if (auto *p2i = isCast(ConversionOp::Ptr2Int, i)) {
+            has_ptr2int = true;
+            // A nonlocal block has a fixed address.
+            has_nondet_op |= !returns_nonlocal(p2i->getValue(), true);
```

Comment:
non-local blocks don't have fixed addresses. They use an UF vs a variable for local blocks, but it's non-deterministic as well, no?

Reply:
Yes, in the twin semantics it is nondeterministic as well.
Here it is opt out because it does not generate quantified variables in that case. Updated the comment